packages/storybook/stories/components/modal/modal.stories.ts

```diff
@@ -14,12 +15,25 @@ const meta: Meta = {
 
 export default meta;
 
+const resetRoot = () => {
+  const storybookRoot = document.querySelector('#storybook-root');
+  if (storybookRoot) {
+    storybookRoot.innerHTML = '';
+  }
+};
+
+const withResetRoot = (storyFn: any) => {
+  resetRoot();
+  return storyFn();
+};
```

Comment:
move `resetRoot` and `withResetRoot` on a dedicated `hook.ts` file (near `control.ts`)